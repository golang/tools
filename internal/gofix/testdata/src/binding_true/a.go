package a

//go:fix inline
func f(x, y int) int { // want f:`goFixInline a.f`
	return y + x
}

func g() {
	f(1, 2) // want `Call of a.f should be inlined`

	f(h(1), h(2)) // want `Call of a.f should be inlined`
}

func h(int) int
