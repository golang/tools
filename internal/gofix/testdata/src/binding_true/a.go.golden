package a

//go:fix inline
func f(x, y int) int { // want f:`goFixInline a.f`
	return y + x
}

func g() {
	_ = 2 + 1 // want `Call of a.f should be inlined`

	var x int = h(1)
	_ = h(2) + x // want `Call of a.f should be inlined`
}

func h(int) int
